services/ogi-route/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const ogiRequest = async (reqUrl: string, rawHeaders: Headers) => {
+  const url = new URL(reqUrl)
+  const { pathname, search } = url
+  const opengraph = `https://ogi.koda.art${pathname}${search}`
+  console.log(url.toString())
+
+  const headers = new Headers(rawHeaders)
+  const request = new Request(opengraph, {
+    headers,
+  })
+
+  return await fetch(request)
+}
```

Comment:
you can use worker to worker communication

Reply:
afaik service bindings only work between worker and worker. meanwhile, ogi is deployed to pages instead of workers